### ISSUE TITLE
Airline theme?

### ISSUE THREAD

**Issue by moshiur-raj** [navarasu/onedark.nvim#6]
onedark.nvim does not provide any theme for vim-airline. Switching to this theme makes airline stick to the default themes and it looks ugly.

**Comment by navarasu** [comment]
Sure. I will add it.

**Comment by moshiur-raj** [comment]
Thank you.

**Comment by navarasu** [comment]
@moshiur-raj Instead of vim-airline. I am thought to support lua only plugins like [hardline](https://github.com/ojroques/nvim-hardline), [bufferline](https://github.com/akinsho/nvim-bufferline.lua).
Your Thoughts?

**Comment by moshiur-raj** [comment]
@navarasu Sounds reasonable to me. I actually was able to fix the problem with airline. airline provides extra themes with [vim-airline-themes](https://github.com/vim-airline/vim-airline-theme). So you don't need to maintain it.
I recently switched to [lualine](https://github.com/hoob3rt/lualine.nvim).

**Comment by baco** [comment]
> @moshiur-raj Instead of vim-airline. I thought to support lua only status plugins like [hardline](https://github.com/ojroques/nvim-hardline), [bufferline](https://github.com/akinsho/nvim-bufferline.lua).
> Your Thoughts?

@navarasu perhaps **also** that, but don't refrain from supporting vim-airlaine. Vim-airline is a widely used plugin, and offers functionality that goes beyond the chosen colorscheme. Many of us have lots of plugins already configured for airline, and having to redo that work for that plugin, just only for the theme, doesn't worth it.

**Comment by moshiur-raj** [comment]
@baco vim-airline already provides a onedark theme. It's available [here](https://github.com/vim-airline/vim-airline-themes).

**Comment by baco** [comment]
Fair enough. Never installed it, because @joshdick's onedark.vim came with it's own implementation for Airline

**Comment by joshdick** [comment]
FWIW, I added the onedark airline theme into vim-airline-themes (in turn based on the one that ships with my own joshdick/onedark.vim) [back in 2017](https://github.com/vim-airline/vim-airline-themes/commits/master/autoload/airline/themes/onedark.vim), but I haven’t kept changes between those two versions synced.

**Comment by baco** [comment]
> FWIW, I added the onedark airline theme into vim-airline-themes (in turn based on the one that ships with my own joshdick/onedark.vim) [back in 2017](https://github.com/vim-airline/vim-airline-themes/commits/master/autoload/airline/themes/onedark.vim), but I haven’t kept changes between those two versions synced.

Will you push an update there, please?

**Comment by navarasu** [comment]
@baco As I support 5 more color variants,  adding a theme to vim-airline-themes will not be good. So I will add airline support in this plugin itself by supporting all color variants. I think it is fine to support few vim plugins until the Lua plugins get mature.

**Comment by baco** [comment]
Thak you @navarasu, that's excellent news!

**Comment by saurabhchardereal** [comment]
Any further update on this?
